test/integration/model/model_actor_light/model.sdf

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<sdf version="1.8">
+
+  <model name="model">
+    <pose>0 0 0 0 0 0</pose>
+    <link name="link">
+
+      <collision name="mesh_col">
```

Comment:
nitpick: Since this just being used for testing the presence of multiple models/lights/actors, the collision and visual don't seem necessary. You do need a link though, but that can be empty.